package.json

```diff
@@ -24,7 +24,6 @@
   "peerDependencies": {
     "code-push": ">=1.8.0-beta",
     "cordova-plugin-file-transfer": ">=1.3.0",
-    "cordova-plugin-file": ">=3.0.0",
```

Comment:
I changed this because `npm install` was giving errors constantly about the installed version of `cordova-plugin-file` not satisfying `cordova-plugin-file-transfer`. Is this the right way to do this?